### PR TITLE
Add expandable-section component

### PR DESCRIPTION
**Pull request by zri5004**
Needs polyfill-focus-visible merged.

### CODE REVIEW

File: packages/expandable-section/src/js/scripts.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        expand.addEventListener('click', () => {
+          content.style['transition-duration'] = (content.scrollHeight / 2) + 'ms';
+          collapse.setAttribute('aria-expanded', 'true');
+          expand.setAttribute('aria-expanded', 'true');
+          cms.expand(content);
+          collapse.style.display = 'block';
+          expand.style.display = 'none';
+
+          // Say something for A/T users.
+          Drupal.announce('expanded', 'assertive');
```
Comment (lal65) [inline packages/expandable-section/src/js/scripts.js:18]: This line is troublesome.  `Drupal.announce` is a non-trivial utility.

We could abstract the `Drupal` away with `cms.announce('...')`, but Pattern Lab / Wordpress would be on their own to implement it.

I was always under the impression that this was not really needed...but Ritu wanted it for whatever reason.

Toggling the `aria-expanded` attribute should be the canonical signal to assistive tech that something has changed.  IMO manually announcing it again is redundant and distracting.

Comment (lal65) [reply packages/expandable-section/src/js/scripts.js:18]: Thoughts on a Wordpress implementation of this being required, @mdw15 ?

Comment (mdw15) [reply packages/expandable-section/src/js/scripts.js:18]: I can implement it in WordPress with your guidance for Next Gen WP Theme.
 
I can add it to the specs.

**Comment by lal65** [comment]
Looks like there's a hidden dependency on CTA buttons here.

I'll get a MR prepped for CTA's...

**Pull request merged**